src/styles/index.scss

```diff
@@ -2,2 +1,134 @@
+@import './variables';
+
+.page-body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  display: flex;
```

Comment:
The `flex-wrap: wrap;` property is not necessary here unless you expect the content to exceed the width of the `.card` and want it to wrap onto a new line. If you don't expect this behavior, you can safely remove this property.